src/components/layout/LeftMenu.vue

```diff
@@ -68,6 +80,9 @@ q-scroll-area.left-menu(style="height: 100%; border-right: 1px solid #ddd")
       :to="item.route"
       ripple
     )
+    ResolveSidebarItem(
+      v-if="isTestnet"
+    )
 </template>
 
 <style lang="sass">
```

Comment:
```suggestion
<style lang="scss">
```
This change also requires translating the code. Again, there are online tools for that.
For example https://codebeautify.org/sass-to-scss-converter